SelfReflection-1-MWoodhouse.qmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Self-Reflection"
+author: "Murphy Woodhouse"
+format: html
+editor: visual
+---
+
+## Questions
+
+**About you**
+
+1.  Please tell me a little bit about your research interests and how you imagine integrating spatial data in your research.
+
+    -   I don't have any projects that could properly be described as research, but I am generally interested in being able to do more sophisticated visualizations for Boise State Public Radio. I am likely to focus on election data for my final project and other exercises.
+
+2.  How do you learn best, and what do you feel is your biggest strength as a student?
+
+    -   I like hands-on learning and immediately putting new skills to work so they firm up. My biggest strength as a student now is that I know what I want to learn and why.
+
+3.  If you want to, you can set a goal for this semester for something you want to work on in the context of this course (outside of the topic matter itself) – for example, using StackOverflow to get answers to analysis questions or using git to track progress on homeworks. If you choose to set this type of goal, what can you do to help support yourself in working toward that goal? How can I help you to meet this goal?
+
+    - I want to get proficient at ingesting, analyzing and visualizing state elections data 
+
+4.  You can, if you choose, also let me know if there is anything else you have going on this semester that you would like me to know about (e.g., field work, other courses, prepping your qualifying exam, working outside of assistantship).
+
+    - I am a full-time employee of Boise State Public Radio, and that is obviously going to be my principal priority. My supervisor is aware that I'm in this course, and happy to accomodate to a degree. However, it is easy to imagine my work getting in the way of super consistent attendance.
```

Comment:
Good to know, and I hope the auditing situation accommodates this! I'm not too familiar with the relationship between graded homework and auditing, so I'll be grading the homeworks you turn in the same as other students. Let me know if this causes you any trouble!